thinkflow/migrations/0003_plantdisease_confidence_plantdisease_disease_name_and_more.py

```
@@ -0,0 +1,48 @@
+            model_name='plantdisease',
+            name='mini_article',
+            field=models.TextField(blank=True, default=''),
+        ),
+        migrations.AddField(
+            model_name='plantdisease',
+            name='preventive_measures',
+            field=models.JSONField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='plantdisease',
+            name='severity',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)